docs/conf.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/master/config
+
+
+# -- Project information -----------------------------------------------------
+
+project = u'CATChem'
+copyright = u'2023, NOAA'
+author = u''
```

Comment:
```suggestion
author = ''
```
The `u`s are not necessary with Python 3, might as well remove them.